user_guide_src/source/incoming/routing/029.php

```diff
@@ -5,6 +5,6 @@
 
 ?>
 
-<!-- Generate the relative URL to link to user ID 15, gallery 12: -->
+<!-- Generate the URI path (route) to link to user ID 15, gallery 12: -->
 <a href="<?= route_to('Galleries::showUserGallery', 15, 12) ?>">View Gallery</a>
```

Comment:
We should never use a route I place of a URL, bad practice.

Reply:
Replaced route_to() with url_to().